src/main/java/Kuso.java

```diff
@@ -0,0 +1,22 @@
+public class Kuso {
+
+    public Kuso() {
+        if (false) {
+            System.out.println("toilet");
+        }
+    }
+
+    public int unko() {
+        String ben = null;
+
+        return Integer.getInteger(ben);
+    }
+
+    public class KusoInner {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vvisteria/danger-spotbugs-sample/blob/9362641e51caa015645900c01d9af8dc09a891d3/src/main/java/Kuso.java#L15"></span>Should Kuso$KusoInner be a <em>static</em> inner class?</td>
    </tr>
  </tbody>
</table>